egs/swbd/s5c/local/reverberate/reverberate_data_dir.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate reverberated data
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse, glob, math, os, random, sys, warnings, copy, imp, ast
+
+nodes = imp.load_source('', 'steps/nnet3/components.py')
+train_lib = imp.load_source('ntl', 'steps/nnet3/nnet3_train_lib.py')
+
+class list_cyclic_iterator:
+  def __init__(self, list, random_seed = 0):
+    self.list_index = 0
+    self.list = list
+    random.seed(random_seed)
+    random.shuffle(self.list)
+
+  def next(self):
+    item = self.list[self.list_index]
+    self.list_index = (self.list_index + 1) % len(self.list)
+    return item
+
+def return_nonempty_lines(lines):
+  new_lines = []
+  for line in lines:
+    if len(line.strip()) > 0:
+      new_lines.append(line.strip())
+
+  return new_lines
+
+
+def GetArgs():
+    # we add compulsary arguments as named arguments for readability
+    parser = argparse.ArgumentParser(description="Writes config files and variables "
+                                                 "for TDNNs creation and training",
+                                     epilog="See steps/nnet3/tdnn/train.sh for example.")
```

Comment:
Add help formatter to the argparse constructor.

e.g.
<pre><code>
3     # we add compulsary arguments as named arguments for readability
 34     parser = argparse.ArgumentParser(description="""
 35     Trains a feed forward DNN acoustic model using the cross-entropy objective.
 36     DNNs include simple DNNs, TDNNs and CNNs.
 37     """,
 38     formatter_class=argparse.ArgumentDefaultsHelpFormatter)

</code></pre>